v3/src/components/mosaic-tile-row.tsx

```diff
@@ -1,10 +1,11 @@
 import { clsx } from "clsx"
 import { observer } from "mobx-react-lite"
-import React from "react"
+import React, { useRef, useState } from "react"
```

Comment:
Since you're not actually implementing resize for mosaic, I would leave this file unchanged -- or perhaps add a comment about cribbing from the `FreeTileRow` resize implementation. Adding code that doesn't work just seems confusing.